### ISSUE TITLE
Design a hex logo

### ISSUE THREAD

**Issue by anyaferguson** [nhs-r-community/NHSRpostcodetools#32]
For the packages page on the NHSR community website (https://nhsrcommunity.com/packages.html), we are using a temporary hex logo for this package, but we would welcome a custom design instead.

Helpful resources that were used for other NHSR packages:

- Original issue: https://github.com/nhs-r-community/nhs-r-community/issues/219
- We used the {hexSticker} package: https://github.com/GuangchuangYu/hexSticker/tree/master
- We used colours from {NHSRtheme}: https://github.com/nhs-r-community/NHSRtheme

**Comment by anyaferguson** [comment]
Hi @francisbarton, nice hex! You may want to update the hex on the NHSR community website too: https://nhsrcommunity.com/r-packages.html.

**Comment by francisbarton** [comment]
Thanks Anya - maybe! I am going to try to get some feedback on the badge informally from DS team soon, let me know if you have any ideas. Before finalising. I have other versions in different colour schemes - I worry that the scarlet and yellow is too stark.

<img width="300" height="346" alt="Image" src="https://github.com/user-attachments/assets/e5d2bd01-b09c-481a-b3ff-1713dca0ccc0" />
<img width="300" height="346" alt="Image" src="https://github.com/user-attachments/assets/511ab20e-c796-46fa-9694-69cafd3027fb" />
<img width="300" height="346" alt="Image" src="https://github.com/user-attachments/assets/ff7215db-108b-4310-96bd-004545419a96" />

Tried to incorporate some [official NHS colours](https://www.england.nhs.uk/nhsidentity/identity-guidelines/colours/). I quite like the purple and aqua green actually.

Jess needs adding to the alt versions of course...

**Comment by anyaferguson** [comment]
Just a quick thought: I wonder how recognisable Postman Pat and Jess are? I like the boundaries drawn in the background, because they make me think of postcodes and maps and are linked to what the package is for. But I'm not sure how many people will get the reference from the characters.

**Comment by francisbarton** [comment]
Yeah not sure. My assumption was "_everyone_ knows Postman Pat" but of course that is not true. Do you think it might come across as exclusionary reference? Or just confusing? I was just trying to think of a way of humanising very abstract things like postcodes via a more visual metaphor or reference. I haven't actually looked yet at the other NHS-R badges you linked to above, maybe there is more of a common style it would be better to adhere to.

Edit: OK just had a look. There is more of a cleaner theme to those, with more NHS blue. Though the border looks very narrow to me compared to what I would expect a hex badge design to have. I guess that is due to the hexsticker package. Allison's badges [here](https://allisonhorst.com/hex-designs) seem to have some thicker and some narrower borders.
I like hex badges to have some character and hopefully humour though if poss. Happy to keep discussing.

**Comment by francisbarton** [comment]
Also my judgement is definitely not at all swayed by how long it took me to make the badge lol 😜😭
